docs/endpoints/configs/ospool.yaml

```diff
@@ -0,0 +1,34 @@
+display_name: OSPool
+engine:
+  type: GlobusComputeEngine
+  max_workers_per_node: 1
+
+  # This config uses apptainer containerization to ensure a consistent
+  # python environment on the worker side. Since apptainer limits writable
+  # directory paths, set working directory paths paths used by the worker to /tmp
+  worker_logdir_root: /tmp/logs
+  working_dir: /tmp/tasks_dir
+
+  # Disable encryption on endpoint-worker communication
+  encrypted: False
```

Comment:
Consider calling out the reasoning for this here as well as "above" in the text.

Reply:
I've added notes in the config.